exporter/datasetexporter/README.md

```diff
@@ -53,8 +64,17 @@ Make sure to provide the appropriate server host value in the `serverHost` attri
   - `retry_max_elapsed_time` (default = 300s): Is the maximum amount of time spent trying to send a buffer.
   - `retry_shutdown_timeout` (default = 30s): The maximum time for which it will try to send data to the DataSet during shutdown. This value should be shorter than container's grace period.
 - `logs`:
-    - `export_scope_info_on_event` (default = false): Include LogRecord scope information (if available) on the DataSet event.
+    - `export_resource_info_on_event` (default = false): Include LogRecord resource information (if available) on the DataSet event.
```

Comment:
From those attributes we are constructing the DataSet event. I will keep it as is. When I add examples, it will be maybe clearer, and then we can adjust the wording.